docs/dev/table/connectors/formats/json.md

```diff
@@ -0,0 +1,201 @@
+CREATE TABLE user_behavior (
+  user_id BIGINT,
+  item_id BIGINT,
+  category_id BIGINT,
+  behavior STRING,
+  ts TIMESTAMP(3)
+) WITH (
+ 'connector' = 'kafka',
+ 'topic' = 'user_behavior',
+ 'properties.bootstrap.servers' = 'localhost:9092',
+ 'properties.group.id' = 'testGroup',
+ 'format' = 'json'
+)
+{% endhighlight %}
+</div>
+</div>
+
+Format Options
+----------------
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left" style="width: 25%">Option</th>
+        <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>format</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what format to use, here should be 'json'.</td>
+    </tr>
+    <tr>
+      <td><h5>fail-on-missing-field</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Flag to specify whether to fail if a field is missing or not, false by default.</td>
+    </tr>
+    <tr>
+      <td><h5>ignore-parse-errors</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Flag to skip fields and rows with parse errors instead of failing;
+      fields are set to null in case of errors, false by default.</td>
+    </tr>
+    </tbody>
+</table>
+
+Data Type Mapping
+----------------
+
+Currently, the JSON schema is always derived from table schema. Explicitly defining an JSON schema is not supported yet.
+
+Flink JSON format uses utility class `com.fasterxml.jackson.databind.node.JsonNodeFactory` (for serialization) and `com.fasterxml.jackson.databind.JsonNode` (for deserialization) provided by `jackson` to convert Flink Data Types to and from JSON data types.
+
+The following table lists the type mapping from Flink type to JSON type.
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left">Flink Data Type</th>
+        <th class="text-center">JSON Data Type</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>NULL</td>
+      <td>null/empty</td>
+    </tr>
+    <tr>
+      <td>CHAR / VARCHAR / STRING</td>
+      <td>string</td>
+    </tr>
+    <tr>
+      <td>BOOLEAN</td>
+      <td>boolean</td>
+    </tr>
+    <tr>
+      <td>BINARY / VARBINARY</td>
+      <td>string</td>
+    </tr>
+    <tr>
+      <td>DECIMAL</td>
+      <td>number</td>
+    </tr>
+    <tr>
+      <td>TINYINT</td>
+      <td>number</td>
+    </tr>
+    <tr>
+      <td>SMALLINT</td>
+      <td>number</td>
+    </tr>
+    <tr>
+      <td>INT/INTERVAL_YEAR_MONTH</td>
+      <td>number</td>
+    </tr>
+    <tr>
+      <td>BIGINT/INTERVAL_DAY_TIME</td>
```

Comment:
Add a new row for interval? `INTERVAL_YEAR_MONTH` and `INTERVAL_DAY_TIME` are root name, not type name. We can call them `INTERVAL` as they all converted to number.